sql/database.sql

```diff
@@ -7640,3 +7640,579 @@ CREATE TABLE `form_clinical_instructions` (
   `activity` TINYINT DEFAULT 1 NULL,
   PRIMARY KEY (`id`)
 )ENGINE=InnoDB;
+
+
```

Comment:
I don't see where you added the additional fields to the user table.

Reply:
I missed that. Have added them.